go/private/repositories.bzl

```diff
@@ -173,9 +172,26 @@ def go_rules_dependencies(force = False):
         patch_args = ["-p1"],
     )
 
+    # gRPC protoc plugin
+    # releaser:upgrade-dep grpc grpc-go
```

Comment:
@fmeum how can I tell the releaser that the `go.mod` is rooted in the `cmd/protoc-gen-go-grpc` subfolder?

Reply:
I do not know off the top of my head, but maybe we'll need to modify the releaser